src/app/pages/questions/components/question/info-screen/info-screen.component.html

```diff
@@ -2,16 +2,30 @@
   class="info-card"
   [class.info-card-sm]="hasFieldLabel"
 >
-  <div *ngIf="isThincItReminder">
-    <img
-      src="assets/imgs/thincIt_app_icon.png"
-      class="icon-thincIt"
-    >
-  </div>
-  <div *ngFor="let item of items">
-    <p class="info-heading">{{ item.heading }}</p>
-    <div class="info-content">
-      <h2>{{ item.content }}</h2>
+  <span
+    item-right
+    (tap)="scrollDown()"
+    class="icon"
+    [class.icon-sm]="hasFieldLabel"
+  >
+    <ion-icon name="arrow-dropdown-circle"></ion-icon>
+  </span>
+
+  <ion-content>
+    <div *ngIf="isThincItReminder">
+      <img
+        src="assets/imgs/thincIt_app_icon.png"
+        class="icon-thincIt"
+      >
```

Comment:
What if this part of the html is contained as one of the info content? Will it be parsed correctly to an icon ? 
Just thinking out loud to make this kind of thing more generic

Reply:
Do you mean in the definition? It will display the HTML string and not the icon. I guess we can implement it similarly to the PDDS range info type description which accepts HTML.

Reply:
Or maybe another property in the definition if they want to show an image?

Reply:
Yes we can think about the best approach for this